modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -266,7 +266,7 @@ impl From<CompressedBlindedMessage> for BlindedMessage {
 
 #[derive(Debug, Clone, Decodable, Encodable)]
 pub enum MintRecoveryState {
-    V0(MintRecoveryStateV0),
+    V1(MintRecoveryStateV0),
```

Comment:
Inconsistent naming

Reply:
It corresponds to the database version ... oh... I guess that doesn't really make sense.